webapp/channels/src/packages/mattermost-redux/src/selectors/entities/scheduled_posts.ts

```diff
@@ -1,10 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import type {ScheduledPost} from '@mattermost/types/schedule_post';
+import type {ScheduledPost, ScheduledPostsState} from '@mattermost/types/schedule_post';
 import type {GlobalState} from '@mattermost/types/store';
 
 import {createSelector} from 'mattermost-redux/selectors/create_selector';
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+
+const emptyList: any[] = [];
```

Comment:
no good practice to type with any[]. Let TS infer it by itself or if you are expecting empty list of X, then do `X[]`(`Post[]`). If you trully need to wait for an unknown type, then use `unknown[]` which at the end will always force you to perform type check before using the variable so hence is safer.

Reply:
correct, there is no reason for this to be any. Fixed by making it a string array.